src/screens/TopicMemberScreen/index.js

```diff
@@ -122,7 +133,10 @@ const TopicMemberScreen = () => {
   const showAnimationHeader = () => {
     interactionManagerRef.current = InteractionManager.runAfterInteractions(() => {
       Animated.timing(animatedHeight, {
-        toValue: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT + dimen.size.TOPIC_FEED_HEADER_HEIGHT,
+        toValue:
+          (coverPath
+            ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+            : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) + dimen.size.TOPIC_FEED_HEADER_HEIGHT,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `showAnimationHeader` function is updated to use the new `animatedHeight` value. The changes seem correct and consistent with the previous hunk.
```diff
- toValue: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT + dimen.size.TOPIC_FEED_HEADER_HEIGHT,
+ toValue:
+   (coverPath
+     ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+     : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) + dimen.size.TOPIC_FEED_HEADER_HEIGHT,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->